src/pyansys_sphinx_theme/__init__.py

```diff
@@ -16,21 +17,22 @@
 
 html_logo = pyansys_logo_black
 
+CSS_FILENAME = "pyansys_sphinx_theme.css"
+
 
 def get_html_theme_path():
     """Return list of HTML theme paths."""
-    theme_path = os.path.abspath(os.path.dirname(__file__))
-    return [theme_path]
+    return Path(os.path.abspath(os.path.dirname(__file__)))
```

Comment:
Since we're going through the trouble using `pathlib.Path`, we might as well use it in lieu of `os`:

```suggestion
    return Path(__file__).parents[0].absolute()))
```

See https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module for a helpful table.